app/models/program_session.rb

```diff
@@ -16,6 +16,10 @@ class ProgramSession < ActiveRecord::Base
 
   validates :event, :session_format, :title, :state, presence: true
 
+  serialize :info, Hash
+
+  attr_accessor :video_url, :slide_url
```

Comment:
So these accessors are unnecessary.  They make a getter and setter (reader and writer) methods which you are hand coding anyway on line 66+.  This line can be deleted safely.